src/sidebar.ts

```diff
@@ -989,13 +1001,15 @@ module.exports = {
       label: "Search",
     },
   ],
-  ...referenceSidebar,
-  ...quickstartSidebar,
-  ...guidesSidebar,
-  ...identitiesSidebar,
-  ...oauth2Sidebar,
-  ...permissionsSidebar,
-  ...selfhostingSidebar,
+  reference: referenceSidebar,
+  "quickstarts-and-tutorials": quickstartSidebar(false),
```

Comment:
And here the unflattened - so all levels - sidebar which is visible after we click on a category.